### ISSUE TITLE
[Bug] No way to setNamespace for a worker to make worker listen for queue in namespace

### ISSUE THREAD

**Issue by oss92** [temporalio/sdk-php#128]
### Describe the bug

There is no way in PHP to ask a worker to listen to specific namespace.

You can create a workflow client as such
```
$workflowClient = WorkflowClient::create(ServiceClient::create($host), $clientOptions->withNamespace($namespace));
```

and then create your commands there which will assign them to a specific namespace.

Creating a worker as such
```
$worker = $factory->newWorker(WorkerFactoryInterface::DEFAULT_TASK_QUEUE, WorkerOptions::new());
```
only listens to the `default` namespace. How can we avoid that?

### To Reproduce

Mentioned in the bug description

### Expected behavior

Somehow in the worker options to set namespace? When I create the workflow client with `default` namespace, everything runs and there are workers listening. If I create the workflow client with a different namespace, I get 

```
There are no Workers currently listening to the Task Queue: default
```

### Screenshots/Terminal output
![Screen Shot 2021-12-01 at 10 51 30](https://user-images.githubusercontent.com/6284234/144212138-4e5712dc-253d-4991-9234-25a175e27dca.png)

### Versions

- OS: Mac
- Temporal Version [e.g. 1.7.0?] latest
- Are you using Docker or Kubernetes or building Temporal from source? Kubernetes

### Additional context
 
Here in go for example: https://github.com/temporalio/samples-go/blob/cac8bb0ee971be97cd2e1f3d48ddfd247a21b7ec/cron/worker/main.go, you can create a client, and pass options which means this worker will listen to this client

**Comment by rustatian** [comment]
Hey @oss92, you can set the namespace via RR config -> https://github.com/spiral/roadrunner-binary/blob/master/.rr.yaml#L154

**Comment by oss92** [comment]
@rustatian I was looking into that and thought so. I think it might be a good idea to edit the generate rr file to include this since it currently does not (when running `./vendor/bin/rr get-binary`)

**Comment by rustatian** [comment]
It will be included in the next release v2.6.1, so, at the moment, please, edit the `.rr.yaml` manually, sorry for that.

**Comment by oss92** [comment]
Yea no problem at all. Just trying to avoid people asking this question again in the future. I really appreciate your quick assistance. Closing this issue.

**Comment by rustatian** [comment]
> Yea no problem at all. Just trying to avoid people asking this question again in the future. I really appreciate your quick assistance. Closing this issue.

You are welcome 👍🏻

**Comment by rustatian** [comment]
> It will be included in the next release v2.6.1, so, at the moment, please, edit the `.rr.yaml` manually, sorry for that.

https://github.com/spiral/roadrunner-binary/releases/tag/v2.6.1 Done.
